snapcraft/wrappers/remote-viewer

```diff
@@ -45,7 +45,7 @@ if [ "${USERNS}" = "0" ]; then
     echo ""
     echo "Your system has unprivileged user namespaces disabled, as a result, "
     echo "you will need to manually run one a SPICE client with the URI above."
-    sleep infinity
+    exec sleep infinity
```

Comment:
Why is this change needed?

Reply:
That's not needed but it caught my eye. The `exec` ensure no `sh` parent sticks around for no purpose because obviously, the `sleep` is terminal (nothing will execute after) and will never return anyway.